openapi_operations.yaml

```diff
@@ -476,6 +476,14 @@ openapi_operations:
     documentation_url: https://docs.github.com/enterprise-server@3.17/rest/enterprise-admin/admin-stats#get-users-statistics
     openapi_files:
       - descriptions/ghes-3.17/ghes-3.17.json
+  - name: POST /enterprises/{enterprise}/access-restrictions/disable
+    documentation_url: https://docs.github.com/enterprise-cloud@latest//rest/enterprise-admin/enterprises#disable-access-restrictions-for-an-enterprise
+    openapi_files:
+      - descriptions/ghec/ghec.json
+  - name: POST /enterprises/{enterprise}/access-restrictions/enable
+    documentation_url: https://docs.github.com/enterprise-cloud@latest//rest/enterprise-admin/enterprises#enable-access-restrictions-for-an-enterprise
+    openapi_files:
+      - descriptions/ghec/ghec.json
```

Comment:
I'm not sure we should change these lines in the PR

Reply:
Well I had to ran `update-openapi`, otherwise the build was failing and these changes were added automatically. I am not going to cherry pick what need to stay and what not.